package.json

```diff
@@ -10,17 +10,36 @@
     "preview": "vite preview"
```

Comment:
- [ ] kindly consider initializing the project with [Create React App](https://create-react-app.dev/)The project should be initialized using [Create React App](https://create-react-app.dev/) not Vite to meet the below requirement
> Project requirements
> Setup a new React application using [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app)

so far I can't continue reviewing your project as it's not set up well kindly set it up well and make sure you meet each point in the project requirements before you as for another review